src/shogun/machine/Machine.cpp

```diff
@@ -176,3 +176,43 @@ std::shared_ptr<LatentLabels> Machine::apply_latent(std::shared_ptr<Features> da
 	error("This machine does not support apply_latent()");
 	return NULL;
 }
+
+std::shared_ptr<Machine> Machine::fit(
```

Comment:
imo these methods shouldnt be added, but this should replace `train` (as discussed on irc with @gf712 )